build_extra/rust/run.py

```diff
@@ -5,6 +5,9 @@
 import sys
 import os
 
-os.environ["OUT_DIR"] = os.path.abspath(".")
-assert os.path.isdir(os.environ["OUT_DIR"])
+# TODO This is for src/msg.rs to know where to find msg_generated.rs
```

Comment:
^-- the comment doesn't bother me, but this is obviously not possible. Cargo does not know where the ninja build dir is and it is never going to. 

Things that are possible -- not suggested to do them now:
* in build.py, `gn gen` and `ninja` inside OUT_DIR, e.g. in "$OUT_DIR/ninja_build". Would be nice for consumers of our crate, would suck for me because then cargo check would build v8 from scratch.
* call flatc from build.rs and write the output to OUT_DIR. Not super great either.

Reply:
> would suck for me because then cargo check would build v8 from scratch.

it's no different than `out/debug`...

Reply:
>  "$OUT_DIR/ninja_build"

Not $OUT_DIR/ninja_build it should write directly into $OUT_DIR

Reply:
> > would suck for me because then cargo check would build v8 from scratch.
> 
> it's no different than `out/debug`...

Would suck for me because I will be using the cargo build for "intellisense" while testing with the gn build.

Reply:
@piscisaureus I'm don't understand. But let's postpone discussion until there's a patch to discuss.

Reply:
> > "$OUT_DIR/ninja_build"
> 
> Not $OUT_DIR/ninja_build - it should write directly into $OUT_DIR

That's a one line change... but ehm lets hold off for a bit actually doing it.